### PR TITLE
[doc] Update readme

### PR DESCRIPTION
**Pull request by SHtress**

### CODE REVIEW

**Review by NikitaZotov — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
-- Install with sc-machine branch **feature/component_manager**
-- To include components-manager kb, add 'sc-machine/sc-tools/sc-component-manager/kb' to platform repo.path file
+## Installation
 
-# Usage
-## Commands
-- components init - downloading specifications from repos.txt sources to _specifications_ folder (automatically created if is not exists)
-- components search [--author \<author\>][--class \<class\>][--explanation \<"explanation"\>]
-- components install [--idtf \<system_idtf\>]
+- Install ostis-web-platform with sc-machine branch **feature/component_manager**
```
Comment (MksmOrlov) [inline README.md:5]: Write about where to change branch (prepare.sh line 47)

File: README.md [hunk 2]
```diff
@@ -1,22 +1,27 @@
...
+- Include sc-component-manager kb, for this add 'sc-machine/sc-tools/sc-component-manager/kb' to platform repo.path file
 
-## Repos structure
-## Repository file
-File repos.txt Contains of two sections: [components] and [repositories].
-[repositories] are links for source (GitHub, google drive etc.) with repos.txt file, [components] are links to source with specification.txt file of component.
+## Usage
```
Comment (MksmOrlov) [inline README.md:8]: Please, write how to run sc-component-manager

File: README.md [hunk 3]
```diff
@@ -1,22 +1,27 @@
...
 
-Example of repository (specifications.scs file)
+### Commands:
+- components init - downloading specifications from repositories. `kb/specifications.scs` contains example of how to describe repository.
+- components search  [--author \<author\>][--class \<class\>][--explanation \<"explanation"\>] - searching component specification in knowledge base. You can search components by author, class or explanation.
```
Comment (MksmOrlov) [inline README.md:12]:
```suggestion
- components search  [--author \<author\>][--class \<class\>][--explanation \<"explanation"\>] - searching component specification in knowledge base. You can search components by author, class or explanation substring.
```

File: src/main.cpp [hunk 4]
```diff
@@ -15,7 +15,7 @@
 #include "sc-memory/utils/sc_signal_handler.hpp"
 
 #include "src/manager/sc_component_manager_impl.hpp"
-#include "src/manager/sc-component-manager-facory/sc_component_manager_factory.hpp"
+#include "src/manager/sc-component-manager-factory/sc_component_manager_factory.hpp"
```
Comment (MksmOrlov) [inline src/main.cpp:18]: Create separate commit or PR for this changes

**Review by MksmOrlov — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (SHtress) [reply README.md:5]: I will make one

Comment (SHtress) [reply README.md:5]: No need if i create branch in ostis-web-platform

File: README.md (hunk 2 above)
Comment (SHtress) [reply README.md:8]: Okay, maybe add script to ostis-web-platform branch

File: src/main.cpp (hunk 4 above)
Comment (SHtress) [reply src/main.cpp:18]: Separate commit

**Review by MksmOrlov — APPROVED** [review]
Make one commit and merge

**Review by KovalM — APPROVED** [review]

**Pull request merged**